server/store/sqlstore/store.go

```diff
@@ -886,17 +886,52 @@ func (s *SQLStore) GetStats() (*storemodels.Stats, error) {
 		return nil, err
 	}
 
-	query = s.getQueryBuilder().Select("count(id)").From("users").Where(sq.NotEq{"RemoteId": ""}).Where(sq.Like{"Username": "msteams_%"}).Where(sq.Eq{"DeleteAt": 0})
+	query = s.getQueryBuilder().Select("count(id)").From("users").Where(sq.And{
```

Comment:
Changed this query to avoid relying on the `msteams_` prefix with a `LIKE` and check for our specific remoteID.